data/property-sort-orders/smacss.txt

```diff
@@ -121,41 +167,66 @@ color
 font
 font-family
 font-size
+font-size-adjust
+font-stretch
+font-effect
+font-smooth
 font-smoothing
+-moz-osx-font-smoothing
+-webkit-font-smoothing
```

Comment:
I'm thinking about deleting **font-smooth** and **font-smoothing** since they aren't standard.